stage.json

```diff
@@ -30,6 +30,11 @@
   "secondary_hero_title": "Extensions are like apps for Firefox.",
   "secondary_hero_summary": "They add features to Firefox to make browsing faster, safer, or just plain fun",
 
+  "static_page_review_guidelines": "https://addons.allizom.org/en-US/review_guide",
+  "static_page_about_firefox_addons": "https://addons.allizom.org/en-US/about",
```

Comment:
these variables are not needed; you can formulate the url in the tests by using {base_url} in an f string.